### ISSUE TITLE
Standard PuTTY is messed up by installation

### ISSUE THREAD

**Issue by alexhorner** [GNS3/gns3-gui#3482]
**Describe the bug**
Installing or updating GNS3 messes with an existing installation of PuTTY, overwriting it's defaults.

**GNS3 version and operating system (please complete the following information):**
 - OS: Windows
 - GNS3 version: 2.2.40.1

**To Reproduce**
1. Install normal PuTTY
2. Use normal PuTTY to see the default
3. Install GNS3
4. Use normal PuTTY and see the difference

**Screenshots or videos**
How it looks (after being messed with by GNS3):
![image](https://github.com/GNS3/gns3-gui/assets/33007665/053a2dde-16cf-4e24-86bb-b94b6eb145e3)

How it should look:
![image](https://github.com/GNS3/gns3-gui/assets/33007665/cdddfe52-7a34-49df-81fe-d149752c0053)

**Comment by grossmj** [comment]
Indeed, the overwrite the PuTTY default settings. One problem is we can't use custom profiles with Solar-PuTTY.

One workaround could be to create a custom profiles in PuTTY and change the putty shortcut to open putty.exe with the `-load` parameter. (untested)

**Comment by alexhorner** [comment]
> Indeed, the overwrite the PuTTY default settings. One problem is we can't use custom profiles with Solar-PuTTY.
> 
> One workaround could be to create a custom profiles in PuTTY and change the putty shortcut to open putty.exe with the `-load` parameter. (untested)

I'd rather GNS3 did this instead of completely screwing up my installation. I use PuTTY a lot, and it was quite a surprise when I launched it to do something mundane and got greeted with a random orange display. I don't even have the default profile anymore, its lost.

**Comment by josephmhiggins** [comment]
@Alex,
I am thinking how to respond to your complaint.
I am a Windows user and it is very, very frustrating when a Windows update comes in and I can not stop it (I can only defer it for 6months) and it is extremely, extremely difficult to know what it does and extremely, extremely difficult to roll it back.
But putty is different.
You **must** have , or **must** gain, the capability and skill to completely configure putty to your specifications in under 15 seconds flat.
Umm, yeah, the gns3 developers might break things sometimes and I have lost my cool - which I should not have done - and I apologize for that. But Windows users should be prepared for anything at anytime breaking GNS3.
But going back to putty, 15 seconds is the limit and it could make you millions of dollars if you configure each and every putty session in 15 seconds.

**Comment by alexhorner** [comment]
> @alex, I am thinking how to respond to your complaint. I am a Windows user and it is very, very frustrating when a Windows update comes in and I can not stop it (I can only defer it for 6months) and it is extremely, extremely difficult to know what it does and extremely, extremely difficult to roll it back. But putty is different. You **must** have , or **must** gain, the capability and skill to completely configure putty to your specifications in under 15 seconds flat. Umm, yeah, the gns3 developers might break things sometimes and I have lost my cool - which I should not have done - and I apologize for that. But Windows users should be prepared for anything at anytime breaking GNS3. But going back to putty, 15 seconds is the limit and it could make you millions of dollars if you configure each and every putty session in 15 seconds.

I'm assuming some of this is missing a /s ? lol.

It's not a massive deal. I'm not particularly angry or upset, just surprised when it happened (and maybe a tad annoyed when I fixed it and a GNS3 update reverted it back) but it is an issue, so as with any issue, I raised it.

As for Windows being a pain, one product being a pain is never an excuse for another product being a pain 😎

**Comment by josephmhiggins** [comment]
So, where are we on your initial complaint?
What do you want the gns3 developers to do?
I want you and need you to be able to resolve a putty misconfigurataion in under 15 seconds
It is very important for you to be able to configure putty in under 15 seconds.

**Comment by alexhorner** [comment]
> So, where are we on your initial complaint? What do you want the gns3 developers to do? I want you and need you to be able to resolve a putty misconfigurataion in under 15 seconds It is very important for you to be able to configure putty in under 15 seconds.

Whether or not I should be able to resolve the issue myself by reconfiguring PuTTY is neither here nor there. And it'll take me a lot longer than 15 seconds to reconfigure all of the colours.

However big or small you think it is, it is still an issue, hence I have raised it.

If I knew what exactly the GNS3 developers would have to do to resolve this, this would be a PR instead of an issue, I am not sure.
